todo/src/serviceWorker.js

```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```

Comment:
Unless you intend on making the app a PWA you can delete the service workers file here